src/main/java/edu/greenblitz/tobyDetermined/commands/rotatingBelly/RotateOutOfDoor.java

```diff
@@ -0,0 +1,24 @@
+package edu.greenblitz.tobyDetermined.commands.rotatingBelly;
+
+import edu.greenblitz.tobyDetermined.RobotMap;
+import edu.wpi.first.wpilibj.Timer;
+
+public class RotateOutOfDoor extends RotatingBellyCommand{
+
+    private static final double endTimeSeconds = 1;
+    @Override
+    public void execute() {
+        belly.setSpeed(RobotMap.RotatingBelly.PERCENTAGE_ROTATING_SPEED);
+    }
+
+    @Override
+    public boolean isFinished() {
```

Comment:
you can remove the time management part from this command and run the command as deadlinecommand with a waitcommad(timeout)